### DISCUSSION TITLE
Welcome to BHBot Discussions!

### DISCUSSION THREAD

**Discussion by ilpersi** [ilpersi/BHBot#186] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by JacsonF** [comment]
First of all thanks for maintaining the project, Is very usefull for the community. I would like to contribute for this project, I don't expert in java, but i liked the ideia for the contributing in something. I forked the project and i'm studing about your functionalities, let me know if you had some tips.
